doc/nrf/device_guides/working_with_nrf/nrf70/developing/sap.rst

```diff
@@ -0,0 +1,123 @@
+.. _nRF70_soft_ap_mode:
+
+SoftAP mode
+###########
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports Software-enabled Access Point ( :abbr:`SAP (Software-enabled Access Point)` ) mode.
+This mode allows a device to operate as a virtual router or temporary access point (AP) and accept connections from other Wi-Fi Station devices.
+SoftAP mode is typically used in scenarios where a Wi-Fi device wishes to share its internet connection with nearby station devices.
+Smartphone Wi-Fi hotspots, for example, utilize the SoftAP mode.
+The smartphone acts as a virtual router, allowing other station devices to connect to the internet through it.
+
+The nRF70 Series devices use SAP mode to support Wi-Fi based provisioning.
```

Comment:
```suggestion
nRF70 Series devices support Wi-Fi provisioning using the SoftAP mode.
```

Reply:
Note to @greg-fer : I hope the sentence above is not too "strong", i.e. it does not imply that SoftAP-based provisioning is the _only_ way to provision nRF70 Series devices.

Reply:
I have added `as one of the provisioning methods.`